index.js

```diff
@@ -11,6 +11,18 @@ function asPromise (fn) {
   });
 }
 
+function isArray (item) {
+  return Array.isArray(item);
+}
+
+function isIORedisFormattedTransactionResults (results) {
```

Comment:
How about a single function

```
function convertToIORedis (results) {
    if Array.isArray(results[0]) {
        return results.map((result) => result[1]);
    }
    return results;
}
```

So below you can do 
```
return asPromise(multi.exec.bind(multi))
    .then(convertToIORedis)  
    .then((results) => {
```

Reply:
That convertToIORedis could have a comment explaining why this convertion is necessary